doc/en/components/grids/_shared/cell-editing.md

```diff
@@ -201,6 +202,97 @@ This code is used in the sample below which implements an [SelectComponent](../s
 
 <!-- end: Angular -->
 
+<!-- Blazor -->
+
+If you want to provide a custom template which will be applied to a cell, you can pass such template either to the cell itself, or to its header. 
```

Comment:
This section is supposed to show how to template the cell editing template, not the header or body templates.

It should be reworked to show how to set the `InlineEditorTemplate`/`InlineEditorTemplateScript` template.